src/middlewared/middlewared/plugins/kerberos.py

```diff
@@ -437,11 +438,10 @@ class Config:
 
     @accepts(
         Int('id', required=True),
-        Dict(
+        Patch(
+            'kerberos_keytab_create',
```

Comment:
Why are you changing this in this PR?

Reply:
I changed this in this PR because i added length validation in `kerberos_keytab_create` schema and would have to do that for update as well, so i instead used a `Patch` as we do elsewhere